frontend/cypress/tests/codeView/codeView_switchAuthorship.js

```diff
@@ -0,0 +1,69 @@
+describe('switch authorship', () => {
+  it('switch authorship view should restore all default controls', () => {
+    Cypress.wait();
+
+    // open the code panel
+    cy.get('.icon-button.fa-code')
+        .should('be.visible')
+        .first()
+        .click();
+
```

Comment:
Hello, I have a question regarding this. Will it be necessary to have a check on the sorting and file types before the switch happens?